README.md

```diff
@@ -487,3 +496,4 @@ You are now able to run `docker-compose up` as usual.
 [jwt.io]: https://jwt.io/#debugger-io
 [Etherpad]: https://github.com/ether/etherpad-lite
 [Jibri]: https://github.com/jitsi/jibri
+[latest release]: https://github.com/jitsi/docker-jitsi-meet/releases/latest
```

Comment:
These change on README.md need to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)